### PR TITLE
Update to OmniSharp 1.36.0

### PR DESCRIPTION
**Pull request by JoeRobich**

### CODE REVIEW

**Comment by JoeRobich** [comment]
GitHub checks did not update but CI was passing - https://travis-ci.org/github/OmniSharp/omnisharp-vscode/builds/716697448

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,66 +9,74 @@
...
+* Improve performance of the `textDocument/codeAction` request. (PR: [omnisharp-roslyn#1814](https://github.com/OmniSharp/omnisharp-roslyn/pull/1814))
+* Provide a warning when the discovered MSBuild version is lower than the minimumMSBuildVersion supported by the configured SDK (PR: [omnisharp-roslyn#1875](https://github.com/OmniSharp/omnisharp-roslyn/pull/1875))
+* Use the real MSBuild product version during discovery (PR: [omnisharp-roslyn#1876](https://github.com/OmniSharp/omnisharp-roslyn/pull/1876))
+* Fixed debugging in .NET 5 preview SDKs ([#3459](https://github.com/OmniSharp/omnisharp-vscode/issues/3459), PR: [omnisharp-roslyn#1862](https://github.com/OmniSharp/omnisharp-roslyn/pull/1862))
 
 ## 1.22.2 (August 04, 2020)
 * Updated Razor support
-  * Improved Semantic Highlighting support by fixing some scenarios which might lead to thrown exceptions and incorrect results. [dotnet/aspnetcore-tooling#2126](https://github.com/dotnet/aspnetcore-tooling/pull/2126)
```
Comment (333fred) [inline CHANGELOG.md:16]: Should these be removed?

Comment (JoeRobich) [reply CHANGELOG.md:16]: reverted. thanks!

**Comment by JoeRobich** [comment]
Again Github checks did not update. CI passing - https://travis-ci.org/github/OmniSharp/omnisharp-vscode/builds/716703443

**Pull request merged**